keyboards/bastardkb/charybdis/3x5/keymaps/via/rules.mk

```diff
@@ -0,0 +1,6 @@
+# Space-saving settings.
+LTO_ENABLE = yes
```

Comment:
Probably should be moved to the keyboard config, as most are likely to run into issues.  And this future proofs it a bit.
```suggestion
```

Reply:
Done.